docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -98,6 +98,10 @@ The following configuration properties are available:
    * - ``http-server.authentication.oauth2.user-mapping.file``
      - File containing rules for mapping user. See :doc:`/security/user-mapping`
        for more information.
+   * - ``http-server.authentication.oauth2.principal-field``
+     - The principal field of access token used for Trino user impersonation.
```

Comment:
of the access token

Reply:
also is it really user impersonation?

Reply:
yes, the field is actually used when creating the Identity of the user. So may be wording is wrong.